docs/DEVELOPING.md

```diff
@@ -39,89 +108,51 @@ source "$NVM_DIR/nvm.sh"
-1. Execute ```scripts/deploy USER@HOSTNAME --mock-ctrl``` or ```scripts/deploy USER@IP_ADDRESS``` replacing USER and HOSTNAME/IP_ADDRESS with the appropriate values for the pi device
-1. Over ssh connection, run ```scripts/run_debug_webserver --mock-ctrl``` from the ```~/amplipi-dev``` directory.
-
-## Installing AmpliPi from scratch on a Pi Compute Module
-1. For a fresh pi compute module, run `scripts/bootstrap-pi`.
-   All pi compute modules shipped with AmpliPi's have already been configured using this script,
-   so this step should not be necessary.
-   Running the script without any arguments will print the instructions.
-   After this step is done SSH is enabled from a fresh Raspberry Pi OS at [amplipi.local].
-1. [amplipi.local] should now be hosted on your network.
+#### Memory Limitation for AmpliPi
+When building directly on the AmpliPi, limit memory usage during the build step:
+
+```sh
+NODE_OPTIONS=--max_old_space_size=768 npm run build
+```
+
+
+
+To develop the frontend, your strategy will bifurcate depending on your access to an Amplipi:
+
+### With an Amplipi
+You will need to edit the `scripts/run_debug_frontend` file with the IP of an Amplipi and then run said file, making sure to revert the file when you're done as to not push your local IP to the repo. This will use that Amplipi's backend so you can live develop the React app on a localhost that dynamically regenerates the app as you save your files. This is the better option, as you can have the localhost app and the `amplipi.local` app in separate tabs to see the differences between what you make and what is already there, particularly if you run a fresh `scripts/deploy` when based on the `main` branch to ensure the system is at its most up-to-date first.
+Please note that live Frontend development has the risk of getting your system into a bad state if you aren't careful with how you handle endpoint interactions or global state saving.
+
+### Without an Amplipi
+Without an Amplipi to use for development, you'll find yourself running everything on the local machine. Follow the above guide on [how to set up `scripts/run_debug_webserver`](#mocked-controller-with-4-stereo-audio-channels-raspberry-pi-os) and then run `scripts/run_debug_frontend` in a separate terminal to run the frontend locally using a simulated backend.
+
+---
+
+## Windows-Specific Setup
+
+### Requirements
+For development on Windows, you’ll need the following tools:
+- Git Bash - This can be achieved in multiple ways, from running inside of WSL or downloading some form of client.
+- [VSCode](https://code.visualstudio.com/) (only recommended, any text editor or IDE will do)
+- [Python 3](https://phoenixnap.com/kb/how-to-install-python-3-windows) (ensure it’s set up in your PATH)
+
+### Notes
+- Windows 10+ supports mDNS for easy SSH access to AmpliPi, but Ethernet is recommended for reliability.
```

Comment:
this sentence doesn't make sense - ethernet and mdns are different layers of the OSI model